hw1Questions.html

```diff
@@ -0,0 +1,21 @@
+2. 	pwd - check current directory
+	mkdir - makes new directory
+	cd - changes to new directory
+	ls - lists the files in a directory
+	help - helps with options 
+
+3. 	git config - configures the username and email for github
+	git clone - makes a copy of the github repository
+	git status - checks the status of repository
+	git add - adds the changes to the repositroy
```

Comment:
Add does not add the changes to the repo. It adds file to the list that will be modified in the next commit. Modifications actually happen when you commit